channeldb/revocation_log.go

```diff
@@ -349,13 +390,31 @@ func deserializeRevocationLog(r io.Reader) (RevocationLog, error) {
 		tlv.MakePrimitiveRecord(
 			revLogCommitTxHashType, &rl.CommitTxHash,
 		),
+		tlv.MakeBigSizeRecord(revLogLocalBalanceType, &localBalance),
+		tlv.MakeBigSizeRecord(
+			revLogRemoteBalanceType, &remoteBalance,
+		),
 	)
+	if err != nil {
+		return rl, err
+	}
 
 	// Read the tlv stream.
-	if err := readTlvStream(r, tlvStream); err != nil {
+	parsedTypes, err := readTlvStream(r, tlvStream)
```

Comment:
curious about what happens when we use the old `readTlvStream`, will we get an EOF error?

Reply:
the old `readTlvStream` doesnt return the parsed types map. So we cant check if the new fields are set or not